README.md

```diff
@@ -3,6 +3,7 @@
 [![CircleCI](https://circleci.com/gh/milesgranger/proffer.svg?style=svg)](https://circleci.com/gh/milesgranger/proffer)
 [![crates.io](http://meritbadge.herokuapp.com/proffer)](https://crates.io/crates/proffer)
 [![License](https://img.shields.io/badge/license-Unlicense-green.svg)](http://unlicense.org/)
+[![Coverage Status](https://coveralls.io/repos/github/milesgranger/proffer/badge.svg?branch=add-code-coverage)](https://coveralls.io/github/milesgranger/proffer?branch=add-code-coverage)
```

Comment:
Going to want to change the branch here to `master`

Reply:
I didn't see that :D